app/src/main/res/layout-v17/clock_fragment.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why you have 2 different clock_fragment layouts? Remove other one, this move to layout folder